src/sas/qtgui/MainWindow/GuiManager.py

```diff
@@ -176,7 +178,7 @@ def addWidgets(self):
         self.results_frame.setVisible(False)
         self.results_panel.windowClosedSignal.connect(lambda: self.results_frame.setVisible(False))
 
-        self._workspace.toolBar.setVisible(LocalConfig.TOOLBAR_SHOW)
+        self._workspace.toolBar.setVisible(config.TOOLBAR_SHOW)
         self._workspace.actionHide_Toolbar.setText("Show Toolbar")
```

Comment:
The toolbar is shown by default, but the test is defaulting to the hidden text. This line should get removed.

Reply:
Don't get what you're saying here.

Reply:
`.setVisible()` sets the visibility and the menu text for the toolbar. Line 182, right after `setVisible()` is called, sets the menu text to the text shown when the toolbar is hidden, regardless of the toolbar visibility. Line 182 should probably be deleted to eliminate the potential inconsistency when the toolbar is shown by default.